README.md

```diff
@@ -32,7 +32,13 @@ apply plugin: 'robolectric'
 
 Add test-only dependencies using the `androidTestCompile` configuration:
 ```groovy
-androidTestCompile 'org.robolectric:robolectric:2.3'
+androidTestCompile('junit:junit:4.12') {
+	exclude module: 'hamcrest-core'
```

Comment:
I get the following `ClassNotFoundException` if I exclude the transitive `hamcrest-core` dependecy: `java.lang.NoClassDefFoundError: org/hamcrest/SelfDescribing`.